analysers/experiments/numberOfProcessInstances.py

```diff
@@ -18,17 +18,25 @@
 def createQuery(dataRDD, experimentID):
```

Comment:
@VincenzoFerme In the trial tables we have process_definition_id = all for the stats on all process instances. The loop in this function will work on that too.

This applies for the other parts you pointed this out too